src/exercise/01.html

```diff
@@ -16,3 +16,20 @@
 <!-- 🐨 Append the div to the root div using `append` -->
 <!--   💰 document.getElementById('root') -->
 <!--   📜 https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/append -->
+
+<html>
+  <body id="body">
+    <script type="module">
+      const root = document.createElement('div');
+      root.id="root";
+
+      const body = document.getElementById('body');
+      body.append(root);
+
+      const div = document.createElement('div');
+      div.textContent = 'Hello World';
```

Comment:
Don't forget that he also wanted us to set the `.className` to 'container' for the div!

Reply:
@benpbolton done :)